amqp.go

```diff
@@ -81,6 +89,12 @@ func NewExchange(url string, options *ExchangeOptions) (*Exchange, error) {
 		return nil, err
 	}
 
+	if options.OnDisconnect == nil {
+		options.OnDisconnect = func() bool {
+			panic("RabbitMQ Disconnected")
+		}
```

Comment:
This is basically the same as the default queue option one above, might as well do this:
``` go
var DefaultOnDisconnect = func() bool {
    panic("RabbitMQ disconnected")
    return false
}
```

and use that in both places.